src/test/java/core/basesyntax/TransactionParserServiceTest.java

```diff
@@ -0,0 +1,79 @@
+package core.basesyntax;
+
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.fail;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.ReaderServiceImpl;
+import core.basesyntax.service.impl.TransactionParserServiceImpl;
+import java.io.File;
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.After;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class TransactionParserServiceTest {
+    private static final int SIZE_OF_DEFAULT_FILE = 8;
+    private static final String INPUT_FILE_HEADER = "operation,fruit,quantity";
+    private static final String DEFAULT_WRONG_OPERATION = "c,apple,25";
+    private static final String DEFAULT_EMPTY_FRUIT = "b,,25";
+    private static final String DEFAULT_QUALITY_IS_WRONG_CHARACTER = "b,apple,k";
+    private static final String FILE_NAME = "src/test/resources/input.csv";
+    private static File testFile;
+    private final TransactionParserServiceImpl transactionParser
+            = new TransactionParserServiceImpl();
+    private final ReaderServiceImpl readerService = new ReaderServiceImpl();
```

Comment:
Please pay attention to [common mistakes](https://mate-academy.github.io/jv-program-common-mistakes/java-core/solid/fruit-shop-tests#ensure-that-you-test-your-services-and-their-independently)